src/runtime_src/core/pcie/linux/device_linux.cpp

```diff
@@ -81,6 +81,44 @@ struct kds_cu_info
   }
 };
 
+/**
+ * qspi write protection status
+ * byte 0:
+ *   0: status not available, 1: status available
```

Comment:
When I first saw this document, I thought the value was 0x00 and 0x01.  Looking at the code shows that these values are actually '0' and '1'.

To help others, please update the comment to reflex characters.  For example:
`'0': status not available, '1': status available`